src/cpp/session/modules/SessionTutorial.cpp

```diff
@@ -249,8 +225,11 @@ void handleTutorialHomeRequest(const http::Request& request,
             << htmlEscape(tutorial.title)
```

Comment:
The heading of each tutorial should actually **be** an HTML heading, probably `<h2>`. Don't want to use `<h1>` because there should generally only be a single `<h1>` in a given page/iframe. In this case we have no `h1` but that's life in the big city.

Reason to use actual heading is that screen readers can then present a list of the tutorials for easier browsing and navigation. Otherwise it's just a big text document with no structure.

Instead of using actual `<h2>` can markup the existing `span` like this:

`<span role="heading" aria-level="2" class="rstudio-tutorials-label">`